content/english/java-basics/activity-7.md

```diff
@@ -1,6 +1,126 @@
 ---
-title: "Data Structure"
+title: "Data Structure (Array & ArrayList)"
 description: "Introduce basic data structures in Java."
 date: 2020-07-16T00:00:00Z
 weight: 8
----
\ No newline at end of file
+---
+In previous exercises, we learned that variables allow us to access and store a piece data. But, what if we want to store a bunch of similiar data? Is there a good way to organize them without declaring multiple variables?
+
+In programming, <b>data structure</b> are structures that allowed coders to store multiple pieces of data of the same type. Using data structure is an efficient way to store and access large pieces of data. In this exercise, we will learn about 2 data structures used in Java: <b>array</b> and <b>ArrayList</b>.
+
+### 1. Array
+Let's say we want to store Patrick's 🐥 top 5 favorite movies. Instead of creating 5 variables, we can store all 5 pieces of data in an array, our first data structure!
+
+To store data in an array, we do the following:
+
+1. We first declare an array by specifying a data type and size (number of data can be stored).
+```java
+String topMovies[] = new String[5]; // create an array of Strings of size 5.
+```
+2. We iterate through the array (go through all the data spots) and assign the data to a spot in the array. 
+
+Note that we access the array with its name and an index (`0` to access the 1st element, `1` for the 2nd ... ).
+```java
+topMovies[0] = "Birds Of Prey";
+topMovies[1] = "The Birds";
+topMovies[2] = "Angry Birds";
+topMovies[3] = "Bird Box";
+topMovies[4] = "The Secret Life of Pets";
+// the array topMovies now store "Birds Of Prey", "The Birds", "Angry Birds", "Bird Box", "The Secret Life of Pets" in order
+```
+
+{{% notice tip %}}
+#### An alternating way to create an array:
```

Comment:
Another alternative way is to use for-loops.

Reply:
To clarify, I would've liked to see an example of initializing an array using a for-loop too, but I guess we already ask the students to do this in the activity.